api/src/controllers/animeController.ts

```diff
@@ -0,0 +1,82 @@
+import type { Request, Response, NextFunction } from '@@types/express.js';
+import { errors, entities } from '@@utils/index.js';
+import { Anime } from '@@entities/index.js';
+
+interface Params {
+    animeId: number
+};
+
+// Create
+async function create(req: Request, res: Response<Params>, next: NextFunction) {
+    const { params } = res.locals;
+    const { animeId } = params as Params;
+
+    const [anime, animeErr] = await entities.insert<Anime>(Anime, req.body);
```

Comment:
You should create a new type for whats expected to be in `req.body` and pass it as a type param to `Request` similar to how you're doing it in `Response<Params>`. You should also check to make sure those expected keys exist on the body, and if they don't send back an `errors.invalidBody()`